app/controllers/summaries_controller.rb

```diff
@@ -3,5 +3,14 @@ def index
     @categories = Category.all
     @date_start = Purchase.minimum(:date)
     @date_end = Date.today
+    @totals = category_totals
```

Comment:
See comments below, but at present `@totals` is the only variable you need - get rid of `@categories` and the date ones.